project/AutoLift.scala

```diff
@@ -1,11 +1,29 @@
 import sbt._
 import sbt.Keys._
 import xerial.sbt.Sonatype._
+import xerial.sbt.Sonatype.SonatypeKeys._
 
 object AutoLift{
 	val ScalaVersion = "2.11.7"
 	val ScalaZ = "7.2.0"
 
+  def module(name: String) =
```

Comment:
Did you think this is a good name? I reused from the readme. In addition the convention that a project named ```x``` is in folder ```x``` is now the default. Maybe we should do the same for the ```build``` definition below.